app/pages/reset-password/new-password-form.spec.js

```diff
@@ -5,15 +5,41 @@ import sinon from 'sinon';
 import NewPasswordForm from './new-password-form';
 
 describe('NewPasswordForm', () => {
+  let wrapper;
+
+  before(() => {
+    wrapper = shallow(<NewPasswordForm />);
+  });
+
   it('renders a new password form for a user', () => {
-    const wrapper = shallow(<NewPasswordForm />);
     assert.equal(wrapper.find('form').length, 1);
   });
 
   it('should call onSubmit when user submits a new password', () => {
     const newPasswordSpy = sinon.spy();
-    const wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
+    wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
     wrapper.find('button').simulate('submit');
     assert.equal(newPasswordSpy.calledOnce, true);
   });
+
+  it('conditionally shows and hides an in-progress icon', () => {
+    wrapper.setProps({ inProgress: true });
+    assert.equal(wrapper.find('i').length, 1);
```

Comment:
Let's be more specific about which `<i>` you're trying to find here: `wrapper.find('i.form-help')`

Enzyme uses a selectors in a very similar way to jQuery, so it'd be good to read up on that.